components/automate-ui/src/app/page-components/event-feed-table/event-feed-table.component.ts

```diff
@@ -107,6 +107,7 @@ export class EventFeedTableComponent implements OnDestroy, OnInit {
   getEventTypeLabel(event: ChefEvent): string {
     const labelMap = {
       version: 'cookbook',
+      cookbook_artifact_version: 'cookbook',
```

Comment:
Cleans up the text in the event feed table
Before
![image](https://user-images.githubusercontent.com/1679247/85806578-4397de80-b704-11ea-8209-ebf2e04691e6.png)
After
![image](https://user-images.githubusercontent.com/1679247/85806538-27943d00-b704-11ea-869c-d8fdd0343d86.png)